On MBSD, ohash is in libutil, but it's not usable.

--- configure.ac.orig	2024-02-23 17:48:20 UTC
+++ configure.ac
@@ -145,10 +145,8 @@ AC_SEARCH_LIBS([imsg_init], [util], [
 	]
 )
 
-AC_SEARCH_LIBS([ohash_init], [util], [], [
 	AC_LIBOBJ(ohash)
 	AC_LIBOBJ(fmt_scaled)
-])
 
 AC_CHECK_FUNCS([asr_run])
 
